api/src/middleware/assessmentsDummyRouter.ts

```diff
@@ -0,0 +1,291 @@
+    }
+
+    let insertedProgramParticipantsRow: {
+      id: number;
+      principal_id?: number;
+      program_id?: number;
+      role_id?: number;
+    };
+
+    try {
+      insertedProgramParticipantsRow = await insertToProgramParticipants(
+        participantIdParsed,
+        programIdParsed,
+        roleId
+      );
+    } catch (error) {
+      next(error);
+      return;
+    }
+
+    res.status(201).json(itemEnvelope(insertedProgramParticipantsRow));
+  }
+);
+
+assessmentsDummyRouter.get(
+  '/submitAssessment/:participantId',
+  async (req, res, next) => {
+    const { participantId } = req.params;
+
+    const participantIdParsed = Number(participantId);
+    if (
+      !Number.isInteger(participantIdParsed) ||
+      participantIdParsed < 1 ||
+      !Number.isInteger
+    ) {
+      next(
+        new BadRequestError(
+          `"${participantIdParsed}" is not a valid participant id.`
+        )
+      );
+      return;
+    }
+
+    const dummyAssessmentSubmissionData = {
+      submission_id: 2,
+      assessment_id: 1,
+      principal_id: participantIdParsed,
+      assessment_submission_state_id: 7, // graded
+      score: 10,
+      opened_at: '2023-02-09 12:00:00',
+      submitted_at: '2023-02-09 13:23:45',
+      responses: [
+        {
+          id: 1,
+          answer_id: 4,
+          assessment_id: 1,
+          submission_id: 2,
+          question_id: 1,
+        },
+        {
+          id: 2,
+          answer_id: 5,
+          assessment_id: 1,
+          submission_id: 2,
+          question_id: 2,
+        },
+        {
+          id: 3,
+          answer_id: 9,
+          assessment_id: 1,
+          submission_id: 2,
+          question_id: 3,
+        },
+        {
+          id: 4,
+          answer_id: 14,
+          assessment_id: 1,
+          submission_id: 2,
+          question_id: 4,
+        },
+        {
+          id: 5,
+          answer_id: 18,
+          assessment_id: 1,
+          submission_id: 2,
+          question_id: 5,
+        },
+        {
+          id: 6,
+          answer_id: 21,
+          assessment_id: 1,
+          submission_id: 2,
+          question_id: 6,
+        },
+        {
+          id: 7,
+          answer_id: 23,
+          assessment_id: 1,
+          submission_id: 2,
+          question_id: 7,
+        },
+        {
+          id: 8,
+          response:
+            'const HelloWorld = () => { return <p>Hello, World!</p>; }; export default HelloWorld;',
+          assessment_id: 1,
+          submission_id: 2,
+          question_id: 8,
+        },
+        {
+          id: 9,
+          response:
+            'React differs from other JavaScript frameworks because it uses a component-based architecture, a virtual DOM, JSX syntax, unidirectional data flow, and is primarily focused on building UIs rather than providing a complete application framework. These features make it faster, more efficient, and more flexible than other frameworks.',
```

Comment:
You can shorten the line by pressing return so that the trailing words are beneath line 146

Reply:
This is okay, I'm approving it for now :) but thank you for the feedback @Paryssatis